package.json

```diff
@@ -35,7 +35,25 @@
     "eject": "react-scripts eject"
   },
   "eslintConfig": {
-    "extends": "react-app"
+    "extends": [
+      "react-app",
+      "plugin:prettier/recommended"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.+(js|jsx)": [
```

Comment:
oh gosh... wtf hahaha

actually this isn't that confusing, it's just referring to all .js .jsx files, right?